src/Swift.Runtime/src/Swift/SwiftString.cs

```diff
@@ -20,14 +21,15 @@ public class SwiftString : IDisposable, ISwiftObject
 {
     private static nuint _payloadSize = SwiftObjectHelper<SwiftString>.GetTypeMetadata().Size;
 
-    public struct Buffer
+    public struct TypeBuffer
```

Comment:
It looks like this is a potentially an ABI type. If so, please add a documentation link.

Reply:
Added, thanks.